src/libawkward/array/ListArray.cpp

```diff
@@ -224,7 +224,7 @@ namespace awkward {
   ListArrayOf<T>::compact_offsets64(bool start_at_zero) const {
     int64_t len = starts_.length();
     Index64 out(len + 1);
-    struct Error err = util::awkward_listarray_compact_offsets64<T>(
+    struct Error err = kernel::listarray_compact_offsets64<T>(
```

Comment:
I don't remember if there was a comment on this one before, but I think this is an instance of me forgetting the underscore; both the kernel dispatch and the underlying kernel should be `..._compact_offsets_64`.